[2023-04-26T08:43:07.866+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-26T08:43:05.048214+00:00 [queued]>
[2023-04-26T08:43:07.872+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-26T08:43:05.048214+00:00 [queued]>
[2023-04-26T08:43:07.872+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T08:43:07.873+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T08:43:07.873+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T08:43:07.880+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-26 08:43:05.048214+00:00
[2023-04-26T08:43:07.885+0000] {standard_task_runner.py:55} INFO - Started process 679 to run task
[2023-04-26T08:43:07.888+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-04-26T08:43:05.048214+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpkdvgfnhe']
[2023-04-26T08:43:07.890+0000] {standard_task_runner.py:83} INFO - Job 209: Subtask is_api_available
[2023-04-26T08:43:07.938+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-04-26T08:43:05.048214+00:00 [running]> on host c452347a64f2
[2023-04-26T08:43:07.982+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T08:43:05.048214+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T08:43:05.048214+00:00
[2023-04-26T08:43:07.984+0000] {http.py:120} INFO - Poking: api/
[2023-04-26T08:43:07.992+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-26T08:43:07.994+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-26T08:43:08.871+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-26T08:43:08.885+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230426T084305, start_date=20230426T084307, end_date=20230426T084308
[2023-04-26T08:43:08.926+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-26T08:43:08.952+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
